pkg/utils/utils.go

```diff
@@ -43,7 +43,7 @@ func BuildCSPDirectives(k8sMode, pluginsCSP, indexPageScriptNonce string) ([]str
 	// When running on-cluster, the default sources are just 'self' (i.e. the same origin).
 	// When running off-cluster, the default sources are 'self' and 'http://localhost:8080' and 'ws://localhost:8080'
 	// (i.e. the same origin and the proxy endpoint).
-	defaultSources := "'self'"
+	defaultSources := "'self' console.redhat.com api.segment.io"
```

Comment:
I don't believe we should be making requests to api.segment.io. We load segment scripts from console.redhat.com. Can we track that down?

Reply:
Yes, I think you're right. I just tracked it down to telemetry configmap we read from console operator on the current dev cluster, and the segment js url is a console.redhat.com address.

Reply:
Yeah, we should try to trace what's making the call to api.segment.io to understand what's happening